AvionicsSoftware.ioc

```diff
@@ -241,8 +241,9 @@ PA7.GPIO_Label=IMU_SPI_MOSI
 PA7.Locked=true
 PA7.Mode=Full_Duplex_Master
 PA7.Signal=SPI1_MOSI
-PA8.GPIOParameters=GPIO_Label
+PA8.GPIOParameters=GPIO_PuPd,GPIO_Label
 PA8.GPIO_Label=BATTERY_EN
+PA8.GPIO_PuPd=GPIO_PULLDOWN
```

Comment:
Something to confirm but this should probably be battery power by default since that should be available always.

I think that is pull up instead of down (could be wrong though)